media/stitcher/api/Stitcher.Samples/GetVodConfig.cs

```diff
@@ -0,0 +1,41 @@
+﻿/*
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START videostitcher_get_vod_config]
+
+using Google.Cloud.Video.Stitcher.V1;
+
+public class GetVodConfigSample
+{
+    public VodConfig GetVodConfig(
+        string projectId, string location, string vodConfigId)
+    {
+        // Create the client.
+        VideoStitcherServiceClient client = VideoStitcherServiceClient.Create();
+
+        GetVodConfigRequest request = new GetVodConfigRequest
+        {
+            VodConfigName = VodConfigName.FromProjectLocationVodConfig(projectId, location, vodConfigId)
+        };
+
+        // Call the API.
+        VodConfig response = client.GetVodConfig(request);

```

Comment:
For simple requests we usually recommend the overload that takes the resource name (instead of creating te request object). I can't remember if we agreed to something else for Video Stitcher?

Reply:
The rest of the samples create a request object. Same for Live Stream API.

Reply:
Acked, that's fine then.